### ISSUE TITLE
Patch for jsdom 11 wrt acorn

### ISSUE THREAD

**Issue by hermanbanken** [jsdom/jsdom#2891]
I know that jsdom 11 is not supported, but hear me out:

Facebook create-react-app uses jest@24.9, jest@24.9 uses jsdom@11 'because updating it might break peoples tests'. Therefore, if you depend on create-react-app, you're stuck with acorn@5.7.4 [which is vulnerable](https://www.npmjs.com/advisories/1488) without a way to resolve this.

Can we create a patch and release jsdom@11.12.1?

```
├── acorn@7.1.1
└─┬ react-scripts@3.4.0
  ├─┬ eslint@6.8.0
  │ └─┬ espree@6.2.0
  │   └── acorn@7.1.1  deduped
  ├─┬ jest@24.9.0
  │ └─┬ jest-cli@24.9.0
  │   └─┬ jest-config@24.9.0
  │     └─┬ jest-environment-jsdom@24.9.0
  │       └─┬ jsdom@11.12.0
  │         └── acorn@5.7.4
  ├─┬ jest-environment-jsdom-fourteen@1.0.1
  │ └─┬ jsdom@14.1.0
  │   ├── acorn@6.4.1
  │   └─┬ acorn-globals@4.3.4
  │     └── acorn@6.4.1
  └─┬ webpack@4.41.5
    └── acorn@6.4.1
```

-----

I'm a bit sad as a result of this dependency hell. I get both react-scripts, jest and jsdom's stance on this. In fact, it wouldn't be an issue if:

1. nobody ever pins patch versions
2. libraries that are vulnerable fix the vulnerability in every minor version they have ever published

Obviously, that is a huge burden, but would especially 2 would really help the JS world forward.

**Comment by hermanbanken** [comment]
```
To: security@npmjs.com
Dear reader,
When reviewing and resolving npm audit issues as a result of advisory https://www.npmjs.com/advisories/1488,
I found that acorn has published a new version 5.7.4 which is fixing the issue, but NPM marks this version as vulnerable.
This results in an unbreakable dependency problem of react-scripts@latest > jest@24.9 > jsdom@11 > acorn@5.7.4.
Neither of those packages is going to update, except acorn, which is wrongly marked as vulnerable.
Regards,
Herman
```

**Comment by domenic** [comment]
Sorry, this is not an exploitable issue in jsdom, _and_ v11 is not supported, so this is not something we'll be acting on. You're welcome to use a fork of either project if that helps you.
